mscore/exportxml.cpp

```diff
@@ -312,10 +313,18 @@ class ExportMusicXml {
       ExportMusicXml(Score* s)
             : _xml(s)
             {
-            _score = s; _tick = {0,1}; div = 1; tenths = 40;
-            millimeters = _score->spatium() * tenths / (10 * DPMM);
+            _score = s;
+            _tick  = Fraction();
+            div    = 1;
+
+            // At the default spatium value of 5pt == 25dots @360dpi,
+            // getTenthsFromPoints and getTenthsFromDots return values @144dpi.
+            // MScore::PPI = 72 = PostScript. 144dpi is exactly double that
+            // resolution, so this scale factor is multiplied by 2.
+            scale  = (s->spatium() / SPATIUM20) * 2;
```

Comment:
I just realized that this formula can be rewritten this way:
`scale = 1.0 / (s->spatium() / (SPATIUM_XML * DPI_F));`
It eliminates the literal value of 2, if that's preferred.  I would change the comment too, to make it clear what the formula is doing.

The other way to do it is to change it from a multiplying factor to a dividing factor, which would also change the `getTenths` functions, but that's not a problem:
`scale = s->spatium() / (SPATIUM_XML * DPI_F);`